recipes/mysql-connector-cpp/all/conanfile.py

```diff
@@ -0,0 +1,49 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake
+
+
+class MysqlConnectorCppConan(ConanFile):
+    name = "mysql-connector-cpp"
+    version = "8.0.27"
+    license = "GPLv2"
+    author = "Oracle"
+    url = "https://dev.mysql.com/doc/connector-cpp/en/"
+    description = "MySQL Connector/C++ is a MySQL database connector for C++."
+    topics = ("mysql", "database", "connector", "c++")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False]}
+    default_options = {"shared": False}
+    generators = "CMakeToolchain"
+    exports_sources = "CMakeLists.txt"
```

Comment:
I can't see this file being part of the pull request - if the library already ahs its `CMakeLists.txt` chances are this is not needed at all